recipe/recipe.yaml

```diff
@@ -0,0 +1,45 @@
+context:
+  name: sentinel
+  version: 1.0.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/sentinel-${{ version }}.tar.gz
+  sha256: 190928f9951af6e94a1f84eefcaed791c28097dd152b88e988906be300451fd2
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - poetry-core
```

Comment:
```suggestion
    - poetry-core >=1.0.0
```

Reply:
Wasn't fast enough :-) I don't think it's a big deal, but just keep an eye on this.

Reply:
ack!